swagger/pelican-swagger.yaml

```diff
@@ -476,7 +476,19 @@ definitions:
       type:
         type: string
         example: "posix"
-        description: "The storage type of the origin server. Is either \"posix\" or \"s3\""
+        description: "The storage type of the origin server. Values can be posix, s3, and https"
```

Comment:
Not a question we need to answer in this PR, but I wonder what our long-term strategy is for enumerating backends in places like this. Ideally we want to add many more backends, but it seems like we list available options manually in a variety of locations. It might be hard to keep things up to date if we don't come up with a solution.

Reply:
Yeah I had the same question. I don't know how we can sync Swagger docs with `parameters.yaml` and other places, but I'd be nice to document what to change for an enum type configuration